en/get-started.md

```diff
@@ -421,7 +437,16 @@ Before proceeding, make sure the following requirements are satisfied:
 
     ``` shell
     kubectl create namespace tidb-cluster
-    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/basic/tidb-cluster.yaml -n tidb-cluster
+    curl -LO https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/basic/tidb-cluster.yaml
+    kubectl -n tidb-cluster apply -f tidb-cluster.yaml
```

Comment:
as suggested by @tennix , we recommend users download the file locally
it's easier for users to know what was applied